context/app/static/js/pages/Organ/Organ.tsx

```diff
@@ -68,15 +85,15 @@ function Organ({ organ }: OrganProps) {
             <Azimuth config={organ.azimuth} />
           </Section>
         )}
-        {shouldDisplaySection[assaysId] && (
+        {shouldDisplaySection[assaysId] && searchHasData && (
           <Section id={assaysId}>
-            <Assays organTerms={organ.search} />
-            <DatasetsBarChart search={organ.search} />
+            <Assays organTerms={searchItems} onHandleSearchStatus={handleDataStatus} />
```

Comment:
I think we can guarantee that if there are datasets, there are samples, but possibly not the other way around.